tests/server/app/test_capabilities.py

```diff
@@ -11,9 +13,17 @@
 
 class CapabilitiesTest(ServerTest):
 
+    def setUp(self) -> None:
+        super().setUp()
+        wait_for_server_startup = os.environ.get('WAIT_FOR_STARTUP',
+                                                 '0') == '1'
+        if wait_for_server_startup:
```

Comment:
Please explain. Why this?

Reply:
The test server is started concurrently to the unit tests. Locally, that's usually not a problem, but on GitHub the tests often fail because of that - the server is not started, but the tests are run anyway. This is a crude but working solution for that. Better ideas appreciated.